Data-Structures/src/main/java/datastructures/tree/FizzBuzzTree.java

```diff
@@ -25,5 +30,31 @@ else if(runnerValue % 3 == 0) {       //check if input is greater than
             replace(runner.right);
         }
     }
+    public ArrayList breadthPrint() {
```

Comment:
Your algorithm looks great, but I don't like the name "breadthPrint" because you are not actually printing anything, you're returning an array list in breadth first order.